.goreleaser.yml

```diff
@@ -16,7 +16,7 @@ builds:
   flags:
   - -trimpath
   ldflags:
-    - -w -X github.com/nats-io/nats-server/v2/server.gitCommit={{.ShortCommit}}
+    - -w -X 'github.com/nats-io/nats-server/v2/server.gitCommit={{.ShortCommit}}' -X 'github.com/nats-io/nats-server/v2/server.serverVersion={{.CurrentTag}}' -X 'github.com/nats-io/nats-server/v2/version.Version={{.CurrentTag}}'
```

Comment:
I see that `serverVersion` makes syft set the version correctly, but do we also need `github.com/nats-io/nats-server/v2/version.Version`?  Doest that also improve other scanners detecting the version?